src/routes/news/components/TunewsList.js

```diff
@@ -48,8 +48,7 @@ class TunewsList extends React.PureComponent<PropsType> {
         <InfiniteScroll
           loadMore={this.loadItems}
           hasMore={!isFetching && hasMore}
-          loader={<LoadingSpinner key={0} />}
-        >
+          loader={<LoadingSpinner key={0} />}>
```

Comment:
Is `key={0}` correct?

Reply:
We just need to set some key here I think :thinking: Although I think it's quite strange that we have to do it, InfiniteScroll should handle this... but oh well